articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -48,7 +54,8 @@ In this example, we will create a host group using 1 availability zone and 2 fau
 1. For **Host group name**, type *myHostGroup*.
 1. For **Location**, select **East US**.
 1. For **Availability Zone**, select **1**.
-1. For **Fault domain count**, select **2**.
+1. Select **Enable Ultra SSD** (Preview) to use Ultra disks with supported Virtual Machines.
```

Comment:
Lowercase ultra

Reply:
This needs to in camel case to align with portal experience.